src/elements/content-explorer/stories/tests/ContentExplorer-visual.stories.js

```diff
@@ -1,25 +1,152 @@
 import { http, HttpResponse } from 'msw';
+import { expect, userEvent, waitFor, within, screen } from '@storybook/test';
 
-import { userEvent, waitFor, within } from '@storybook/test';
 import ContentExplorer from '../../ContentExplorer';
-import { DEFAULT_HOSTNAME_API } from '../../../../constants';
 import mockRootFolder from '../__mocks__/mockRootFolder';
-import { defaultVisualConfig, SLEEP_TIMEOUT } from '../../../../utils/storybook';
+import mockSubFolder from '../__mocks__/mockSubFolder';
+import { defaultVisualConfig } from '../../../../utils/storybook';
+
+import { DEFAULT_HOSTNAME_API } from '../../../../constants';
+
+export const basic = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
 
-export const basic = {};
+        expect(await canvas.findByText('An Ordered Folder')).toBeInTheDocument();
+        expect(canvas.getByText('Tue Apr 16 2019 by Jeremy Press')).toBeInTheDocument();
+
+        expect(canvas.getByText('Book Sample.pdf')).toBeInTheDocument();
+        expect(canvas.getByText('Thu Dec 8 2022 by Jeremy Press')).toBeInTheDocument();
```

Comment:
jeremy is an ex-boxer. i don't think we should codify his name in the repo. can we use one of the docs modifed by "Preview"?